### PR TITLE
Update/smartmon playbook

### PR DESCRIPTION
**Pull request by technowhizz**
Ensure that cron is installed before attempting to configure crontab

### CODE REVIEW

File: etc/kayobe/ansible/smartmon-tools.yml [hunk 1]
```diff
@@ -23,6 +23,12 @@
         - nvmemon.sh
       become: yes
 
+    - name: Ensure cron is installed
+      ansible.builtin.package:
+        name: cron
```
Comment (jovial) [inline etc/kayobe/ansible/smartmon-tools.yml]: Is the package `cronie` on RedHat?

Comment (technowhizz) [reply etc/kayobe/ansible/smartmon-tools.yml]: Yeah it is. Thank you!

File: etc/kayobe/ansible/smartmon-tools.yml [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
 
   tasks:
-    - name: Ensure smartmon-tools and nvme-cli is installed
+    - name: Ensure smartmon-tools, jq, nvme-cli and cron/cronie is installed
       package:
         name:
           - smartmontools
           - nvme-cli
           - jq
+          - "{{ 'cron' if ansible_facts['distribution'] == 'Ubuntu' else 'cronie' }}"
```
Comment (Alex-Welsh) [inline etc/kayobe/ansible/smartmon-tools.yml:11]: At what point will this be run? will you always have access to ansible_facts? could you use os_distribution?

Comment (jovial) [inline etc/kayobe/ansible/smartmon-tools.yml:11]: nit: using ansible_facts['os_family'] == 'Debian' might be more generic, but given that that we only support Ubuntu/Redhat this looks OK.

**Review by jovial — APPROVED** [review]

File: etc/kayobe/ansible/smartmon-tools.yml (hunk 2 above)
Comment (technowhizz) [reply etc/kayobe/ansible/smartmon-tools.yml:11]: we're not setting gather_facts: false so facts should be available

File: etc/kayobe/ansible/smartmon-tools.yml [hunk 3]
```diff
@@ -2,12 +2,13 @@
 - hosts: overcloud
 
   tasks:
-    - name: Ensure smartmon-tools and nvme-cli is installed
+    - name: Ensure smartmon-tools, jq, nvme-cli and cron/cronie is installed
```
Comment (markgoddard) [inline etc/kayobe/ansible/smartmon-tools.yml:5]: This is why I usually just put "packages" or "package dependencies" :)

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/ansible/smartmon-tools.yml (hunk 3 above)
Comment (technowhizz) [reply etc/kayobe/ansible/smartmon-tools.yml:5]: Ah. Will bear in mind for next time

**Pull request merged**